tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/DockerContext.ts

```diff
@@ -0,0 +1,102 @@
+import { Logger } from "winston";
+import { DockerCliConfig } from "../schema/dockerCliConfig";
+import { initializeLogger } from "@azure-tools/sdk-generation-lib";
+import path from "path";
+import fs from "fs";
+import { sdkToRepoMap } from "./generateCodesInLocal";
+
+export class DockerContext {
+    mode: 'generateCodesInLocal' | 'growUp' | 'generateCodesInPipeline';
+    readmeMdPath?: string;
+    tag?: string;
+    sdk: string[];
```

Comment:
is this for single SDK or the list of SDKs?  Maybe it is better to call sdkList?

Reply:
sure, will change.